packages/imp/imp.dev2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "imp"
+synopsis: "Experimental library using modular implicits"
+maintainer: "yallop@gmail.com"
+authors: ["Leo White" "Jeremy Yallop"]
+homepage: "https://github.com/modular-implicits/imp"
+dev-repo: "git+https://github.com/modular-implicits/imp.git"
+bug-reports: "https://github.com/modular-implicits/imp/issues"
+license: "MIT"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+   "dune" {build}
+   "ocaml-variants"
+     { = "4.02.1+modular-implicits" | = "4.02.1+modular-implicits-ber" }
+]
+
+url {
+  src: "https://github.com/modular-implicits/imp/archive/3701b83ed210cf04af4dea6a6d9dc2e9babf302e.tar.gz"
```

Comment:
This would be a good point to (git-)tag with a semantic version number, and use a URL that points to the tag